UQ-eLIPSE/crucible
# Story/vethub 9

Distinguish between admin and a general user on client side.

To test:
Go to https://crucible-uat.uqcloud.net as a general user (no record in the database).
Check there is no edit functionality available.
Add your username to the db and the functionality should appear.
Run the current cypress test to check current resource creation is still functioning.
